src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/email/ProcessedImagesEmailBuilder.java

```diff
@@ -110,6 +103,8 @@ private JSONArray generateAllTasksJsons(StringBuilder errorBuilder) {
 				} catch (JSONException e1) {
 					LOGGER.error("Failed to create UNAVAILABLE task json.", e);
 				}
+			} catch (Exception e) {
+				e.printStackTrace();
```

Comment:
printStackTrace does not help us debug. Should we log instead?

Reply:
I updated exception catchers of this method.